config/database.rb

```diff
@@ -1,13 +1,35 @@
 configure do
-  # Log queries to STDOUT in development
-  if Sinatra::Application.development?
+
+  case Sinatra::Application.environment
+  # Development database settings
+  when :development
+    # Log queries to STDOUT in development
     ActiveRecord::Base.logger = Logger.new(STDOUT)
-  end
+    set :database, {
+      adapter: "sqlite3",
+      database: "db/db.sqlite3"
```

Comment:
Is it useful anymore or just confusing to call this one `db.sqlite3` when the rest are named after the environment?